Loop/Plugins/LoopPlugins.swift

```diff
@@ -67,6 +67,43 @@ class PluginManager {
 
         })
     }
+    
+    func getCGMManagerTypeByIdentifier(_ identifier: String) -> CGMManagerUI.Type? {
+        for bundle in pluginBundles {
+            if let name = bundle.object(forInfoDictionaryKey: LoopPluginBundleKey.cgmManagerIdentifier.rawValue) as? String, name == identifier {
+                do {
+                    try bundle.loadAndReturnError()
+                    
+                    if let principalClass = bundle.principalClass as? NSObject.Type {
+                        
+                        if let plugin = principalClass.init() as? LoopUIPlugin {
+                            return plugin.cgmManagerType
+                        } else {
+                            fatalError("PrincipalClass does not conform to LoopUIPlugin")
+                        }
+                        
+                    } else {
+                        fatalError("PrincipalClass not found")
+                    }
+                } catch let error {
+                    print(error)
```

Comment:
[nit] For debugging or permanent?

Reply:
Again follow pump manager pattern (line 52). But I have been using this for debug.